setup.py

```diff
@@ -122,6 +122,9 @@ def _format_pulp_requirement(plugin, specifier=None, ref=None, gh_namespace="pul
     "social-auth-app-django>=3.1.0,<4.0.0",
     "dynaconf>=3.1.12",
     "django-auth-ldap==4.0.0",
+    "distro",
+    "boto3",
+    "insights_analytics_collector@git+https://git@github.com/RedHatInsights/insights-analytics-collector.git@main#egg=insights_analytics_collector"
```

Comment:
will you need new release of the insights_analytics_collector? Talk to Martin

Reply:
yes, any dependency here will need to come from pypi rather than github prior to merge

@dmzoneill we'll also want a dependency range specified here, something like `>=1.2.3,<2.0.0`

Reply:
added dependency range